README.md

```diff
@@ -44,7 +44,7 @@ git log "$PREVIOUS_TAG"..HEAD
 # check the diff between the previous tag and HEAD
 git diff "$PREVIOUS_TAG"
 # if that all looks good, tag
-DATE=`date +%Y%m%d%H%M%S`
+DATE=`date --utc +%Y%m%d%H%M%S`
```

Comment:
Hm, this doesn't work on OSX... looks like we want `TZ=UTC date +%Y%m%d%H%M%S` there.

If TZ envvar works on linux, we can use that.  Otherwise we may have to have an `if $platform` thing?